### ISSUE TITLE
convert_to_output fails to detach torch tensors before returning dataframe

### ISSUE THREAD

**Issue by djinnome** [ciemss/pyciemss#350]

**Comment by djinnome** [comment]
Line 90

https://github.com/ciemss/pyciemss/blob/78141de11968d36a4536697a1ca60714f4b42313/src/pyciemss/utils/interface_utils.py#L90

Should be:

```python
                f"{observable_id}_obs": torch.squeeze( expression(**expression_vars)).detach().cpu().numpy().astype(np.float64)
```
